xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-macro/src/main/resources/ApplicationResources.properties

```diff
@@ -114,4 +116,7 @@ livedata.displayer.boolean.false=False
 livedata.displayer.xObjectProperty.missingDocumentName.errorMessage=Can't save an XObject property with a missing document name.
 livedata.displayer.xObjectProperty.failedToRetrieveField.errorMessage=Failed to retrieve the {0} field.
 
-livedata.filter.list.emptyLabel=---
\ No newline at end of file
+livedata.filter.list.emptyLabel=---
+
+livedata.footnotes.computedTitle=Some pages have a computed title. Filtering and sorting by title will not work as expected for these pages.
+livedata.footnotes.propertyNotViewable=Displayed when some pages require special rights to be viewed and thus fields cannot be extracted from them.
```

Comment:
Unlike the previous message, which is specific to page title, this one is generic, independent of the live data source: the live data configuration allows you to express that a given entry is not viewable. So we should:

* either rephrase it to refer to **entries** rather than **pages**
* or add support for "overwriting" the message with a more specific one "from the live data source"

I'd keep it simple an rephrase the message.